Automattic/themes
# Added header padding when there is no menu

#### Changes proposed in this Pull Request:
There was no padding value assigned to the header when there was not a
menu available.  This change uses the same padding values as are used
WITH a menu for when there is none.

#### Testing
* Install Spearhead theme.
* Setup a menu and assign it to 'Primary Navigation'
* Note the location of the title (in the header).  (This is the expected location)
* Remove the 'Primary Navigation' menu (via the customizer)
* Note the location of the title.
* The expected location is the same location as when there is a 'Primary
Navigation' menu assigned.

Menu:
![image](https://user-images.githubusercontent.com/146530/97323360-1b052a00-1847-11eb-9bb2-5f50f8832d09.png)

No Menu:
![image](https://user-images.githubusercontent.com/146530/97323457-2fe1bd80-1847-11eb-9794-3e1cb3dfae34.png)

Mobile Menu:
![image](https://user-images.githubusercontent.com/146530/97323628-60295c00-1847-11eb-8be5-d10c6e9d7456.png)

Mobile No Menu:
![image](https://user-images.githubusercontent.com/146530/97323559-4ee04f80-1847-11eb-8215-32116cdec558.png)


#### Related issue(s):
Fixes #2697
